### DISCUSSION TITLE
How do I remove a wrapper?

### DISCUSSION THREAD

**Discussion by griiid** [The-Wineskin-Project/WineskinServer#350] [Q&A] [answered]
Hello, I've created a wrapper, and I want to remove it, but I can't, how to do it?

**Comment by emendelson** [comment]
Can't you simply drag it to the trash? If you mean something other than a "wrapper", then look in `~/Library/Application Support/Wineskin`

**Comment by Gcenx** [comment]
Wrappers you've created are store in `~/Applications/Wineskin` so just trash the wrapper in that directory.

**Comment by griiid** [comment, marked as answer]
The problem I met is:
I just delete the wrapper from Finder, but it still displays in the Launch Pad; When I click-and-hold on it, there's not close (x) button appeared.

I finally found that **I should delete it from the Trash**, and then click it in the Launch Pad again, it will know the item was totally deleted, and then I can do click-and-hold on it, and there's a close (x) button.
Thank you.
